themes/src/main/resources-community/theme/base/account/messages/messages_de.properties

```diff
@@ -133,15 +150,20 @@ federatedIdentityLinkNotActiveMessage=Diese Identit\u00E4t ist nicht mehr aktiv.
 federatedIdentityRemovingLastProviderMessage=Sie k\u00F6nnen den letzten Eintrag nicht entfernen, da Sie kein Passwort haben.
 identityProviderRedirectErrorMessage=Fehler bei der Weiterleitung zum Identity Provider.
 identityProviderRemovedMessage=Identity Provider erfolgreich entfernt.
+identityProviderAlreadyLinkedMessage=Die f\u00F6derierte Identit\u00E4t von {0} ist bereits einem anderen Benutzer zugewiesen.
+staleCodeAccountMessage=Die Seite ist abgelaufen. Bitte versuchen Sie es noch einmal.
```

Comment:
"Seite abgelaufen" sounds a bit strange.

Reply:
Vielleicht "Diese Seite ist nicht mehr gültig, bitte versuchen Sie es noch einmal"

Reply:
Vielleicht "Diese Seite ist nicht mehr gültig, bitte versuchen Sie es noch einmal"

Reply:
I like Thomas' suggestion and am going to pick that.